### ISSUE TITLE
 Uncaught Error: Call to undefined method Redux_Helpers::is_theme()

### ISSUE THREAD

**Issue by banderson623** [awesomemotive/wp-mail-logging#132]
I had to disable our use of this plugin, but here is the error we got when hosting on wordpress.com, after an auto upgrade.

An error of type E_ERROR was caused in line 28 of the file /srv/htdocs/wp-content/plugins/wp-mail-logging/src/inc/redux/WPML_Redux_Framework_config.php. Error message: Uncaught Error: Call to undefined method Redux_Helpers::is_theme() in /srv/htdocs/wp-content/plugins/wp-mail-logging/src/inc/redux/WPML_Redux_Framework_config.php:28

Stack trace:
```#0 /srv/htdocs/wp-content/plugins/wp-mail-logging/src/WPML_Init.php(111): No3x\WPML\Settings\WPML_Redux_Framework_config->__construct(Array)
#1 /srv/htdocs/wp-content/plugins/wp-mail-logging/lib/vendor/pimple/pimple/src/Container.php(113): No3x\WPML\WPML_Init->No3x\WPML\{closure}(Object(No3x\WPML\WPML_DI_Container))
#2 /srv/htdocs/wp-content/plugins/wp-mail-logging/src/WPML_DI_Container.php(12): No3x\WPML\Pimple\Container->offsetGet('redux')
#3 /srv/htdocs/wp-content/plugins/wp-mail-logging/src/WPML_Init.php(125): No3x\WPML\WPML_DI_Container->addActionsAndFilters()
#4 /srv/htdocs/wp-content/plugins/wp-mail-logging/wp-mail-logging.php(106): No3x\WPML\WPML_Init->init('/srv/htdocs/wp-...')
#5 /wordpress/core/5.8.2/wp-settings.php(409): include_once('/srv/htdocs/wp-...')
#6 /srv/htdocs/wp-config.php(81): require_once('/wordpress/core...')
#7 /wordpress/core/5.8.2/wp-load.php(55): require_once('/srv/htdocs/wp-...')
#8 /wordpress/core/5.8.2/wp-login.php(12): require('/wordpress/core...')
#9 {main}
```

**Comment by flakas** [comment]
Hello @banderson623,
Thanks for the issue report.
I've just released a new plugin version 1.10.1 that should fix this issue.
Could you please confirm if the issue persists? Thanks!

**Comment by capuderg** [comment]
Redux Framework was removed in the latest version: 1.11.0.
